ui/components/multichain/avatar-group/avatar-group.tsx

```diff
@@ -44,22 +51,38 @@ export const AvatarGroup: React.FC<AvatarGroupProps> = ({
       gap={1}
     >
       <Box display={Display.Flex}>
-        {visibleMembers.map((member, i) => (
-          <Box
-            borderRadius={BorderRadius.full}
-            key={member.symbol}
-            style={
-              i === 0 ? { marginLeft: '0' } : { marginLeft: marginLeftValue }
-            }
-          >
-            <AvatarToken
-              src={member.iconUrl}
-              name={member.symbol}
-              size={size}
-              borderColor={borderColor}
-            />
-          </Box>
-        ))}
+        {visibleMembers.map((member, i) => {
+          const avatarType = member.avatarType || AvatarType.TOKEN;
```

Comment:
I don't think `avatarType` should be on `member` level, it should be at component usage level (a prop) because we're never going to mix TOKEN and ACCOUNT types.